courses/free-courses-hi.md

```diff
@@ -6,6 +6,7 @@
 * [Bash and Shell](#bash-and-shell)
 * [C](#c)
 * [C++](#cpp)
+* [Computer Networks](#computer-networks)
```

Comment:
Other lists use "Networking" and "Network". Unless you have a specific reason, using one of those will helb make search work better

Reply:
Reason i used "Computer Networks" is because most of the Universities in India(as the folder is for hindi speakers) use the  term "Computer Network" instead of "Network". If you will still want me to change "Computer Network" to Networking, i will do it without any hesitation.